ImputationPipeline/PerformPopulationPCA.wdl

```diff
@@ -396,12 +404,12 @@ task SubsetToArrayVCF {
     Array[File] intervals
     Array[File] intervals_index
     String basename 
-    Int disk_size = 3*ceil(size([vcf, intervals, vcf_index], "GB")) + 20
+    Int disk_size = 3 * ceil(size([vcf, intervals, vcf_index], "GB")) + 300
```

Comment:
I believe it would still be cheaper than a long-running job crashing and having to restart with more disk space, though I agree that it would be nicer to set this number in a more "clever" way. I'll leave this the way it is for now but open to suggestions.